### PR TITLE
Association users ideas

### PR DESCRIPTION
**Pull request by korneliacieslik**

### CODE REVIEW

File: app/controllers/ideas_controller.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class IdeasController < ApplicationController
   def index
-    @ideas = Idea.all
+    @ideas = current_user.ideas
```
Comment (nidalaa) [inline app/controllers/ideas_controller.rb]: ❤️

File: app/models/idea.rb [hunk 2]
```diff
@@ -1,2 +1,3 @@
 class Idea < ApplicationRecord
+	belongs_to :user
```
Comment (nidalaa) [inline app/models/idea.rb]: Indentation looks a bit strange, is it two spaces or two tabs?
Try to change your settings in Sublime https://stackoverflow.com/a/22704309

File: db/schema.rb [hunk 3]
```diff
@@ -21,6 +21,8 @@
     t.string "picture"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.bigint "user_id"
+    t.index ["user_id"], name: "index_ideas_on_user_id"
```
Comment (nidalaa) [inline db/schema.rb]: 👏  for index! 🎉

File: app/controllers/ideas_controller.rb [hunk 4]
```diff
@@ -33,6 +33,9 @@ def show
   private
 
   def idea_params
-    params.require(:idea).permit(:name, :description, :picture)
+    params
+      .require(:idea)
+      .permit(:name, :description, :picture)
+      .merge(user: current_user)
```
Comment (nidalaa) [inline app/controllers/ideas_controller.rb]: I's be a bit more explicit with current user and use 

current_user.ideas.new (idea params) in `new` action 🤔 .

However, there is nothing wrong with this approach, so take it or leave it :)

**Review by nidalaa — CHANGES_REQUESTED** [review]

File: app/controllers/ideas_controller.rb (hunk 4 above)
Comment (wmaciejak) [reply app/controllers/ideas_controller.rb]: Really good point! @korneliacieslik would you mind to take a look ? :)

Comment (korneliacieslik) [reply app/controllers/ideas_controller.rb]: @nidalaa  it is fixed now. Could you take a look?

File: app/models/idea.rb (hunk 2 above)
Comment (korneliacieslik) [reply app/models/idea.rb]: It's fixed

**Comment by nidalaa** [comment]
@korneliacieslik LGTM, ready to merge once you resolve conflicts 👍 
Please hit Merge button once it is done! I tryst you here! :shipit:

**Pull request merged**